packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.base.tsx

```diff
@@ -188,4 +197,25 @@ export class PersonaCoinBase extends BaseComponent<IPersonaCoinProps, IPersonaSt
 
     this.props.onPhotoLoadingStateChange && this.props.onPhotoLoadingStateChange(loadState);
   };
+
+  private _onCoinMouseEnter = () => {
+    if (this.showInitialsOnHover()) {
+      this.setState({
+        isHovered: true
+      });
+    }
+  };
+
+  private _onCoinMouseLeave = () => {
+    if (this.showInitialsOnHover()) {
+      this.setState({
+        isHovered: false
+      });
+    }
+  };
+
+  // Due to showInitialsOnHover having two possible places to reside, check both and return if it is set to true in either place
+  private showInitialsOnHover = () => {
+    return Boolean(this.props.showInitialsOnHover || (this.props.coinProps && this.props.coinProps.showInitialsOnHover));
```

Comment:
We shouldnt need to pass down showInitialsOnHover in normal props and coinprops

Reply:
This was to deal with the edge case in facepile where it renders a persona instead of a persona coin whenever there is only 1 persona being passed in. We couldn't directly pass in showInitialsOnHover to it and have to pass it back through the coinProps prop.

Reply:
Hm, so there is an implicit dependency on this prop being passed down in that case? Sounds a bit fragile. Can we solve that more explicitly?

Reply:
Ah, misread, it is defined in sharedprops, so its not implicit. But this means there are basically two ways of passing this down, which feels bad. Can we clean that up somehow?

Reply:
We could add showInitialsOnHover back to sharedPersonaProps but that would ultimately only be used in PersonaCoin itself. Another alternative would be to uncouple persona from facepile so it only uses PersonaCoin. This would not be backwards compatible unless we use a Boolean flag to force the facepile to use a PersonaCoin instead of a Persona whenever we want.

Reply:
Cant the personacoin props be passed down explicitly from facepile.props -> persona.coinprops -> personacoin.props?

Reply:
By this do you mean having showInitialsOnHover being a part of the IPersonaCoinProps but assuming it is stored in coinProps on IPersonaSharedProps. Then only checking coinProps for it within PersonaCoin?

Reply:
Yes :)
Might look a bit messier though